libcrux-psq/src/ecdh_binder.rs

```diff
@@ -0,0 +1,310 @@
+//! # ECDH Binder
+//!
+//! This module implements a protocol binder that binds a post-quantum
+//! pre-shared key component to an outer ECDH based key agreement.
```

Comment:
The comments read like this could be any ECDH key, but it is implement for x25519 only.
That's fine, but we should mention it at least once up here.